### ISSUE TITLE
Question about italian trust list endpoint

### ISSUE THREAD

**Issue by FrancescoBonizzi** [DevTrevi/DgcReader#28]
Hi! I noticed a strange behaviour, not with your library, but in general when you make an http get request to `https://get.dgc.gov.it/v1/dgc/signercertificate/status`.

Please, consider the following code:
```
var httpClient = new HttpClient();
var response = await httpClient.GetAsync("https://get.dgc.gov.it/v1/dgc/signercertificate/status");
var body = await response.Content.ReadAsStringAsync();
```

I am on Windows **11**, and I run it from these .NET framework versions
- .NET Core 3.1: it works
- .NET 4.8: it doesn't work
- .NET 6: it doesn't work

When it doesn't work, the error is **Cannot estabilish a secure SSL/TLS channel**:
![image](https://user-images.githubusercontent.com/11348479/147059822-71bc3027-e341-4b93-9bb7-39ba38925fba.png)

Also, when you make the request with browser, it works.

I investigated it a little and I found these things:
- The server wants a TLS3.1 handshake
- Windows 10 doesn't implement it
- Windows 11 should implement it
- Chromium implements his own version of TLS3.1 that's why it works even if in the OS it's not implemented
- If a publish a web app with that code on Azure, it works

But I really don't understand why in .NET Core 3.1 it works and with the next .NET versions not.

I'd like to know you opinion because I'm very curious about it!

Thanks!

**Comment by bortolin** [comment]
Try read this [https://docs.microsoft.com/it-it/dotnet/framework/network-programming/tls](https://docs.microsoft.com/it-it/dotnet/framework/network-programming/tls).

**Comment by DevTrevi** [comment]
Hi @FrancescoBonizzi , I did some testing and it seems that there is some sort of regression in Windows 11 about the TLS handshaking. The Italian trustlist endpoint requires a Tls1.2 (tested with Postman)

As posted by @bortolin , the best practice is to leave the ServicePointManager.SecurityProtocol to "SystemDefault" in order to let the OS choose which protocol to use.
This works fine in Windows 10 when targeting .NET Framework >= 4.7 and with .NET / .NET Core >= 3.1

The same is not true for Windows 11, where .NET Framework >=7 fails when using SystemDefault and only works by setting Tls12 explicitly in ServicepointManager.SecurityProtocol

.NET Core 3.1 works as expected, but .net5 and .net6 fails.
I found a temporary solution on [this issue](https://github.com/serilog-contrib/serilog-sinks-elasticsearch/issues/399) related to Serilog. It requires setting a registry key in order to disable the 1.3 TLS protocol by default:
```
[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client]
"DisabledByDefault"=dword:00000001
```

After adding this key I was able to execute the request using .net5 and .net6, and also using SystemDefault with .NET Framework >= 4.7
